package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Mon Nov 23 09:54:34 UTC 2015 - mfilka@suse.com
+
+- bnc#956012
+  - routing configuration exported at the end of first stage even
+    when configured via linuxrc
```

Comment:
It is unclear whether this describes the bug or the fix :) Also, can you please mention the user visible symptoms?